### DISCUSSION TITLE
Add an isometric hex grid to the editor.

### DISCUSSION THREAD

**Discussion by sstari** [ppy/osu#23539] [Ideas] [answered]
This would be cool to add, as I, and others often enjoy mapping using this grid type, but they are a pain to create in both osu!stable editors and osu!lazer editors. This would be a huge motivator for me to move onto the lazer editor as it'd be much more convenient!

**Comment by bdach** [comment, marked as answer]
See https://github.com/ppy/osu/issues/8773
